pymobiledevice3/services/mobile_image_mounter.py

```diff
@@ -338,23 +340,31 @@ def auto_mount_developer(
     image_mounter.mount(image_path, signature)
 
 
-async def auto_mount_personalized(lockdown: LockdownServiceProvider) -> None:
+async def auto_mount_personalized(lockdown: LockdownServiceProvider, version: Optional[str] = None) -> None:
```

Comment:
```suggestion
async def auto_mount_personalized(lockdown: LockdownServiceProvider, version: str = LATEST_DDI_BUILD_ID) -> None:
```
That way we don't need to check for it explicitly